docs/checkit.js

```diff
@@ -783,6 +783,89 @@ CheckIt.prototype.findCurrentDay = function() {
 };
 
 
+// Should these be CheckIt functions? or Calendar. TODO
+// How to access the calendar object when I need it for these functions?
+// current active calendar...??
+
+CheckIt.prototype.getNumberOfChecked = function() {
+    // Returns the number of checked days in a calendar
+
+    var checkit = this;
+
+    return checkit.store.getActive().then(function(activeCalendarId) {
+        console.log("found this activeCalendarId: " + activeCalendarId);
+        return checkit.store.loadById(activeCalendarId).catch(function(err) {
+            console.log("Couldn't find active calendar specifically in getNumberOfChecked");
```

Comment:
They aren't really errors. I was just using console.log to figure out what was happening when I was running the function and what wasn't happening. But I will change to console.error since those pop up more anyway.